.github/workflows/build-and-push.yml

```diff
@@ -44,3 +61,4 @@ jobs:
           docker_context: ${{ matrix.dockerfile_path }}
           dockerfile_path: ${{ matrix.dockerfile_path }}
           tag: ${{ matrix.tag }}
+          suffix: ${{ matrix.suffix }}
```

Comment:
```suggestion
          suffix: ${{ matrix.tag }}
```

Reply:
Please not, when we will support Fedora build-and-push, the `tag` will not be used.

Reply:
Or better say, the suffix will not be used at all.